### PR TITLE
feat!: Bump libraries to support Sepolia

### PR DESCRIPTION
**Pull request by lauti7**

### CODE REVIEW

File: src/lib/peer.ts [hunk 1]
```diff
@@ -1,15 +1,16 @@
-import { LambdasClient } from 'dcl-catalyst-client/dist/LambdasClient'
+import { createLambdasClient, LambdasClient } from 'dcl-catalyst-client/dist/client/LambdasClient'
 import { Profile } from '../modules/profile/types'
 import { BaseAPI } from './api'
 import { FetchProfileOptions, ProfileEntity } from './types'
+import { createFetchComponent } from '@well-known-components/fetch-component'
```
Comment (meelrossi) [inline src/lib/peer.ts:5]: can you move this import up?

File: src/lib/peer.ts [hunk 2]
```diff
@@ -27,10 +28,8 @@ export class PeerAPI extends BaseAPI {
     }
 
     this.cache[address] = this.lambdasClient
-      .fetchProfiles([address.toLowerCase()], {
-        headers: { 'Cache-Control': 'max-age=0' }
-      })
-      .then(profiles => profiles[0])
+      .getAvatarsDetailsByPost({ ids: [address.toLowerCase()] })
```
Comment (meelrossi) [inline src/lib/peer.ts:31]: is the cache control no needed anymore?

File: src/modules/profile/sagas.spec.ts [hunk 3]
```diff
@@ -115,7 +115,7 @@ describe('when handling the action to set the profile avatar description', () =>
 
 describe('when handling the action to set the profile avatar alias', () => {
   const alias = 'anAlias'
-
+
```
Comment (meelrossi) [inline src/modules/profile/sagas.spec.ts:118]: can you remove this space 😬

File: src/lib/peer.ts (hunk 2 above)
Comment (lauti7) [reply src/lib/peer.ts:31]: I think that as the request is done by the POST method, the request is not cached by the browser, so no need of adding the cache control header. Besides that, the new function doesn't accept sending headers

**Review by fzavalia — APPROVED** [review]
Given that dependencies had major updates, release this as a major change. To do so, update the merge commit name to something like explained in semantic-release docs https://github.com/semantic-release/semantic-release#commit-message-format

**Comment by fzavalia** [comment]
I've updated the PR name so it is deployed as a mayor 😄

**Pull request merged**